test/IntegrationTests/StrongNamedTests.cs

```diff
@@ -0,0 +1,60 @@
+// <copyright file="StrongNamedTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using IntegrationTests.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace IntegrationTests;
+
+public class StrongNamedTests : TestHelper
```

Comment:
Even though we only have strong naming for .net framework, I'm assuming that we want this test to also apply to newer versions of .net?

Reply:
The test also runs against `netcoreapp3.1` and `net6.0`.